global-news-app/src/Charts/ChartsSection/PihChart/PihChart.tsx

```diff
@@ -0,0 +1,81 @@
+import ChartCard from "../ChartCard/ChartCard";
+import EmptyState from "components/EmptyState/EmptyState";
+import { ChartsProps } from "../ChartsSection";
+import { PihChartContainer } from "./PihChart.style";
+import { PieChart, Pie, Cell } from "recharts";
+import { Charts } from "Types/enums";
+import Legend from "./Legend/Legend";
+import { COLORS } from "Utils/consts";
+import { isEmpty } from "lodash";
+
+interface ISourceCounts {
+  [name: string]: number;
```

Comment:
you use this type several times, so you can create a type inside the types.ts.
use the Record utility